praw/models/reddit/subreddit.py

```diff
@@ -2072,38 +2096,28 @@ class SubredditWiki(object):
 
     def __getitem__(self, page_name):
         """Lazily return the WikiPage for the subreddit named ``page_name``.
-
```

Comment:
Can you remove these whitespace changes?

Reply:
I don't remember putting these in, odd...